app/views/candidate_mailer/application_choice_submitted.text.erb

```diff
@@ -1,9 +1,9 @@
 Hello <%= @application_form.first_name %>
 
-
-  You have submitted an application for <%= @application_choice.course_option.course.name_and_code %> at <%= @application_choice.course_option.course.provider.name %>.
+You have submitted an application for <%= @application_choice.course_option.course.name_and_code %> at <%= @application_choice.course_option.course.provider.name %>.
 
 [Sign in to your account to check the progress of your application](<%= @candidate_magic_link %>).
 
+Your training provider will contact you if they would like to organise an interview.
 
-  Your training provider will contact you if they would like to organise an interview.
+<%= render 'get_help_teacher_training_adviser' %>.
```

Comment:
Is there an extra full stop here?

<img width="498" alt="image" src="https://github.com/user-attachments/assets/39e5c0cc-aaa3-492d-8aaa-faaedc99c3f5">

https://apply-review-10063.test.teacherservices.cloud/rails/mailers/candidate_mailer/application_choice_submitted

Reply:
Good catch!